circle.yml

```diff
@@ -0,0 +1,26 @@
+# CircleCI is only used for Ansible-Lint and Ansible Syntax checks
+# CircleCI default image is Ubuntu 14.04 w/ Python 2.7 (same for all Data.gov instances)
+# Test Kitchen (WIP) should be used for Ansible Playbook test provisioning
+dependencies:
+  override:
+    - |
+      sudo apt-add-repository -y ppa:ansible/ansible
+      sudo apt-get update
+      sudo apt-get install -y ansible
+    - sudo apt-get update && sudo apt-get install git
+    - sudo pip install ansible-lint
+    - sudo ansible-galaxy install -r ./tests/requirements.yml --ignore-errors
+    - cp tests/secrets-example.yml tests/group_vars/all/secrets.yml
+#  post:
+#    - ansible-lint --version
+
+test:
+  override:
+  # Ansible syntax check
+  - sudo ansible-playbook --syntax-check:
+      files:
+        - tests/test.yml
+  # Ansible lint check
+  - sudo ansible-lint -v:
+      files:
+        - tests/test.yml
```

Comment:
I would steal the `Makefile`/`test.override` configuration from https://github.com/GSA/jenkins-deploy/compare/master...8637ea509a2a68dd56bab369a1618ba97515951e, minus the `./test/run.sh` part, since the Terraform validation and Ansible linting were working there.